chucks-pick3-web/angular-app/src/app/services/form-validation.service/form-validation.service.ts

```diff
@@ -56,6 +56,57 @@ export class FormValidationService {
     }
     return alphabetical;
   }
+  public isLowercase(input: string): boolean {
+    let lowercase = false;
+    const alphaChars = 'abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ'.split('');
```

Comment:
No need for upper case characters if you are only checking for lower case characters.  Simplify.

Reply:
Simplified alphaChars in form validation.